### PR TITLE
Seed data in the mongodb database

### PR DESCRIPTION
**Pull request by chrichri17**
## What does this PR do

Seed dummy data in mongodb using json files.

__Usage__:
```bash
(venv) $ python3 manage.py seed --mode=refresh
```

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #73** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/73" target="_blank">Graphite</a>  👈
    * **PR #74** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/74" target="_blank">Graphite</a> 
      * **PR #77** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/77" target="_blank">Graphite</a> 
        * **PR #78** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/78" target="_blank">Graphite</a> 
          * **PR #79** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/79" target="_blank">Graphite</a> 
      * **PR #75** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/75" target="_blank">Graphite</a> 
        * **PR #76** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/76" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/73?utm_source=stack-comment).

File: backend/api/management/commands/seed.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+from pathlib import PosixPath
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
 
-from backend.api.client import client
+from backend.api.client import CollectionName, client
 
 # USAGE :
 #   python manage.py seed --mode=(refresh | clear)
```
Comment (YacineSteeve) [inline backend/api/management/commands/seed.py:11]: Maybe you should mention that `clear` is the default mode.

**Review by YacineSteeve — APPROVED** [review]
Awesome 💯 

What if you also add a target for this in `Makefile` ?
Something like `make seed`

File: backend/api/management/commands/seed.py (hunk 1 above)
Comment (chrichri17) [reply backend/api/management/commands/seed.py:11]: Yep. But honestly, `clear` is not useful right now haha. I should remove it...

Comment (chrichri17) [reply backend/api/management/commands/seed.py:11]: it should just be `python3 manage.py seed`

Comment (YacineSteeve) [reply backend/api/management/commands/seed.py:11]: Hum, okay. I'm not against that

**Pull request merged**